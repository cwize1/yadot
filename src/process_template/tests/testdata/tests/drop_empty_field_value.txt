cat: dog
elephant:
    ${{ drop }}: rat
