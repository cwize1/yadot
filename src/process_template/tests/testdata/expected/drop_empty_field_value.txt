ERROR: <None>
OUTPUT:
---
cat: dog
elephant: {}